includes/Modules/Idea_Hub.php

```diff
@@ -965,4 +967,24 @@ function( $idea ) use ( $ideas_with_posts ) {
 		return array_values( $ideas );
 	}
 
+	/**
+	 * Escapes and formats the message shown as notices.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $message The message to escape and format.
+	 * @return string Escaped notice message.
+	 */
+	private function escape_notice_content( $message ) {
+		$message = wp_kses(
+			$message,
+			array(
+				'a' => array(
+					'href' => array(),
+				),
+			)
+		);
+
+		return '<p>' . $message . '</p>';
```

Comment:
@asvinb this is a side effect for this function. We probably need to rename it to something like `escape_and_wrap_notice_content` to better reflect what this method does. Or we can also make this method be an anonymous function in the `admin_notice_idea_hub_ideas` method. It can be done something like this:

```php
private function admin_notice_idea_hub_ideas( $notices ) {
    ...

    $escape_and_wrap_notice_content = function( $message ) {
        $message = wp_kses( ... );
        return $message;
    };
    
    ...
    
    $notices[] = new Notice( self::SLUG_SAVED_IDEAS, array(
        'content'         => function() use ( $escape_and_wrap_notice_content ) {
            $message = sprintf( ... );
            return $escape_and_wrap_notice_content( $message );
        },
        ...
    ) );
    
    ...
}
```

Reply:
Nice. Updated! I guess am not sure when to use an anonymous function in PHP or just have it as a method in the class? 🤔

Reply:
We don't have specific rules for anonymous functions in PHP. In general, if it is a small ad-hoc function for a specific method, then we can use an anonymous function.